baseline/client/tests/n-back.test.js

```diff
@@ -4,7 +4,7 @@ import { pressKey, cartesianProduct } from "./utils.js"
 describe("n-back", () => {
     it("results should have at least one result marked isRelevant", () => {
         // check timeline nodes
-        const timeline = (new NBack(1)).getTimeline();
```

Comment:
Out of curiosity, why did this change? Shouldn't set 1 also have isRelevant stuff in it?

Reply:
Ah, never mind - I didn't see the commit comment when I was looking at this with all of the other changes.